index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <header class="main-nav">
+            <div class="main-logo"><a href="#">Oscar Byiringiro</a></div>
+            <nav>   
+                <div class="bottom-icons">
+                    <img src="./images/Union.png">
+                        
+                </div>
+            </nav>
+        </header>
+        <section class="main-body">
+            <div class="container">
+                <div class="intro">
+                    <h1>Hey There <br> I'm Oscar</h1>
+                </div>
+                <div class="title">
+                    <h3>I'am a software developer</h3>
+                </div>
+                <div class="description">
+                    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        don’t hestiate to contact me.</p>
+                </div>
+                <div class="logo">
+                    <ul>
+                        <li><a href="#"><img src="./images/twitter.png"></a></li>
+                        <li><a href="#"><img src="./images/linkedin.png"></a></li>
+                        <li><a href="#"><img src="./images/hand.png"></a></li>
+                        <li><a href="#"><img src="./images/twitter.png"></a></li>
+                        <li><a href="#"><img src="./images/m.png"></a></li>
+                    </ul>
+                    
+                    
+                    
+                    
+
+                                        
+                                    
+                                                                              
+                </div>
+            <div>
+                    
+                </div>
```

Comment:
- [ ] Kindly remove this unused `div` as pointed out by the previous reviewer, It will help make the codebase cleaner and readable. 😎️